x-pack/plugins/enterprise_search/public/applications/test_utils/mock_kibana_context.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { httpServiceMock } from '../../../../../../src/core/public/mocks';
```

Comment:
For some reason tslint yells when I attempt to do `'src/core/public/mocks'`, not totally sure why 😞